extensions/nodes/node_status.go

```diff
@@ -116,11 +117,11 @@ func AllMachineReady(client *rancher.Client, clusterID string, timeout time.Dura
 	return err
 }
 
-// AllNodeDeleted is a helper method that will loop and check if the node is deleted in the cluster.
+// AllNodeDeleted is a helper method that will loop and check if all of the node is deleted in the cluster.
 func AllNodeDeleted(client *rancher.Client, ClusterID string) error {
 	ctx := context.Background()
 	err := wait.PollUntilContextTimeout(
-		ctx, oneSecondInterval, fiveMinuteTimeout, true, func(ctx context.Context) (bool, error) {
+		ctx, oneSecondInterval, defaults.ThirtyMinuteTimeout, true, func(ctx context.Context) (bool, error) {
 			nodes, err := client.Management.Node.ListAll(&types.ListOpts{
```

Comment:
just noting that this is using management. Is there a way to update this to v1 instead? Or is this function only for rke1 (if so could that be noted in the godoc comment)?

Reply:
There probably is, but then I would want to update the `node_status.go` as a whole to us v1. That's out the scope here in this issue, so I'd want to do that in a separate PR.

Reply:
Talked offline, will do this in a separate PR.